hugo-site/content/information/heritage.md

```diff
@@ -0,0 +1,75 @@
+      * Automatic failover to one of two different graphical drivers ("vesa" or "scfb") should the main graphics driver fail to work (such as installing the wrong version of the NVIDIA driver for your system).
+      * Dynamic loading of the video driver and any associated kernel modules on bootup, allowing the system harddrive to be moved between systems as needed with little to no loss of functionality (provided that the drivers for both are installed already - otherwise one of the failover drivers may be used).
+* Graphical utilities for common tasks
+   * The PCDM login manager.
+   * The [Lumina Desktop Environment](https://lumina-desktop.org) is pre-installed and pre-configured for out-of-box usage.
+   * A [graphical system-update utility](https://github.com/project-trident/trident-updatemgr) has been written just for Project Trident. This provides easy access to configuring and performing updates in addition to the ability to roll-back a system update as desired.
+   * Removable media detection and on-demand usage is provided via integrations between Project Trident and the Lumina Desktop.
+* Stable OS, Rolling Ports
+   * Project Trident uses the 6-month "stable" branch of TrueOS for the OS-components themselves. This ensures there are no radical changes or binary compatibility issues for a 6-month period of time.
+   * Project Trident also builds and provides a package repository of every "port" contained in the TrueOS ports repository. This package repository is also updated every week, ensuring a steady stream of new or updated applications very soon after they appear in FreeBSD/TrueOS.
+   * Project Trident uses a [customized build configuration](https://github.com/project-trident/trident-build/blob/master/trident-master.json/) for both OS and ports to ensure that everything is built specifically to be used for a graphical operating system.
+
+## Differences From FreeBSD
+
+Many of these differences from FreeBSD actually come from the TrueOS project, so this list will change over time as the TrueOS and FreeBSD projects continue to evolve.
+
+### OpenRC
+
+TrueOS provides the OpenRC system for managing services on the OS in place of the "legacy" RC system currently used on FreeBSD. This difference is most notable in a few key ways:
+
+* Faster bootup - OpenRC starts up services much faster and with better dependency handling than FreeBSD.
+* Better status reporting - the `rc-status` utility provides an instant look of all the services on your system and their current status.
+* Service files are located in "[/usr/local]/etc/init.d" rather than "[/usr/local]/etc/rc.d"
+* Some services have different names: "dhcpcd" instead of "dhclient", "network" instead of "netif"
+* Many OpenRC services are "multiplexed" for individual device/profile management
+   * Example: The "network.wlan0" service controls the first wifi device, and should be started/stopped instead of the "network" service when reconfiguring the wifi settings.
```

Comment:
```suggestion
   * Example: The **network.wlan0** service controls the first wifi device, and should be started/stopped instead of the **network** service when re-configuring the Wi-Fi settings.
```